api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -102,4 +104,20 @@ public GroupsApi groupApi(@Qualifier(END_USER_API_CLIENT) ApiClient apiClient) {
     return api;
   }
 
+  @Bean
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public StatusApi statusApi(WorkbenchConfig workbenchConfig) {
+    StatusApi statusApi = new StatusApi();
+    ApiClient apiClient = new ApiClient();
```

Comment:
So is the deal that we don't need to be auth'd to call the status endpoint, and thus want to make a separate ApiClient here?